pkg/utils/exec.go

```diff
@@ -103,7 +103,10 @@ func (c *Cmd) RoundTripCallback(conn *websocket.Conn) (*http.Response, error) {
 	wg.Wait()
 	close(errChan)
 	err := <-errChan
-	return nil, err
+	return &http.Response{
```

Comment:
This was needed because gcloud plugin reads the response (and was causing a `panic`)